asistente_ladm_col/gui/dialogs/dlg_quality.py

```diff
@@ -84,120 +79,22 @@ def __init__(self, db, qgis_utils, quality, parent=None):
-                }, {
-                    'id': 'check_boundaries_covered_by_plots',
-                    'text': translated_strings[CHECK_BOUNDARIES_COVERED_BY_PLOTS]
-                }, {
-                    'id': 'check_boundary_nodes_covered_by_boundary_points',
-                    'text': translated_strings[CHECK_BOUNDARY_NODES_COVERED_BY_BOUNDARY_POINTS]
-                }, {
-                    'id': 'check_dangles_in_boundaries',
-                    'text': translated_strings[CHECK_DANGLES_IN_BOUNDARIES]
-                }]
-            }
-        self.items_dict[QCoreApplication.translate("QualityDialog", "Rules for Polygons")] = {
-                'icon': 'polygons',
-                'rules': [{
-                    'id': 'check_overlaps_in_plots',
-                    'text': translated_strings[CHECK_OVERLAPS_IN_PLOTS]
-                },{
-                    'id': 'check_overlaps_in_buildings',
-                    'text': translated_strings[CHECK_OVERLAPS_IN_BUILDINGS]
-                },{
-                    'id': 'check_overlaps_in_rights_of_way',
-                    'text': translated_strings[CHECK_OVERLAPS_IN_RIGHTS_OF_WAY]
-                },{
-                    'id': 'check_plots_covered_by_boundaries',
-                    'text': translated_strings[CHECK_PLOTS_COVERED_BY_BOUNDARIES]
-                #}, {
-                #    'id': 'check_missing_survey_points_in_buildings',
-                #    'text': QCoreApplication.translate("QualityDialog", "Buildings nodes should be covered by Survey Points")
-                }, {
-                    'id': 'check_right_of_way_overlaps_buildings',
-                    'text': translated_strings[CHECK_RIGHT_OF_WAY_OVERLAPS_BUILDINGS]
-                }, {
-                    'id': 'check_gaps_in_plots',
-                    'text': translated_strings[CHECK_GAPS_IN_PLOTS]
-                }, {
-                    'id': 'check_multipart_in_right_of_way',
-                    'text': translated_strings[CHECK_MULTIPART_IN_RIGHT_OF_WAY]
-                }, {
-                    'id': 'check_plot_nodes_covered_by_boundary_points',
-                    'text': translated_strings[CHECK_PLOT_NODES_COVERED_BY_BOUNDARY_POINTS]
-                },{
-                    'id': 'check_buildings_should_be_within_plots',
-                    'text': translated_strings[CHECK_BUILDING_WITHIN_PLOTS]
-                },{
-                    'id': 'check_building_units_should_be_within_plots',
-                    'text': translated_strings[CHECK_BUILDING_UNIT_WITHIN_PLOTS]
-                }]
+        for enum_quality_rule_group, quality_rule_group_name in self.quality_rules_manager.quality_rule_groups.items():
+            self.items_dict[quality_rule_group_name] = {
+                'rules': [{'id': k_rule, 'text': v_rule.rule_name} for k_rule, v_rule in self.quality_rules_manager.get_quality_rules_by_group(enum_quality_rule_group).items()]
```

Comment:
Simplify this double for loop...
It seems there is no need for `quality_rule_groups`.